pgx/src/callbacks.rs

```diff
@@ -328,7 +328,7 @@ where
                 for hook in hooks.iter() {
                     let hook = hook.borrow();
                     if let Some(hook) = hook.as_ref() {
-                        crate::guard::guard(|| (hook.0)(my_subid, parent_subid));
+                        pg_sys::guard(|| (hook.0)(my_subid, parent_subid));
```

Comment:
This will longjmp in the case of errors over a bunch of rust stuff, which will probably be leaked at best. I think what we want here is this whole fn to have the guard

Reply:
That thought had crossed my mind.  I'm glad you pointed it out.